PyYAML: Getting requirements to build wheel did not run successfully

**Environment information**
* MWDB version (from `/about`): 2.9.0
* Installation method:
  - [ ] mwdb.cert.pl service
  - [ ] From PyPi (`pip install mwdb-core`)
  - [x] From docker-compose
  - [ ] Other (please explain)
* Plugins installed:
None 

**Behaviour the bug (what happened?)**

Upon running `docker-compose up` the build fails stating that the wheel build failed.

Error message:
```
Collecting PyYAML==5.4
  Downloading PyYAML-5.4.tar.gz (174 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 174.8/174.8 kB 16.5 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [62 lines of output]
      running egg_info
      writing lib3/PyYAML.egg-info/PKG-INFO
      writing dependency_links to lib3/PyYAML.egg-info/dependency_links.txt
      writing top-level names to lib3/PyYAML.egg-info/top_level.txt
      /tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/config/setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
      !!
      
              ********************************************************************************
              The license_file parameter is deprecated, use license_files instead.
      
              By 2023-Oct-30, you need to update your project and remove deprecated calls
              or your builds will no longer be supported.
      
              See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
              ********************************************************************************
      
      !!
        parsed = self.parsers.get(option_name, lambda x: x)(value)
      Traceback (most recent call last):
        File "/app/venv/lib/python3.8/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/app/venv/lib/python3.8/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/app/venv/lib/python3.8/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 341, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 323, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 338, in run_setup
          exec(code, locals())
        File "<string>", line 271, in <module>
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 107, in setup
          return distutils.core.setup(**attrs)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 185, in setup
          return run_commands(dist)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
          dist.run_commands()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 969, in run_commands
          self.run_command(cmd)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 1234, in run_command
          super().run_command(command)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/dist.py", line 988, in run_command
          cmd_obj.run()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 314, in run
          self.find_sources()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 322, in find_sources
          mm.run()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 551, in run
          self.add_defaults()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 589, in add_defaults
          sdist.add_defaults(self)
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/command/sdist.py", line 104, in add_defaults
          super().add_defaults()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/command/sdist.py", line 251, in add_defaults
          self._add_defaults_ext()
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/command/sdist.py", line 336, in _add_defaults_ext
          self.filelist.extend(build_ext.get_source_files())
        File "<string>", line 201, in get_source_files
        File "/tmp/pip-build-env-a2966zd7/overlay/lib/python3.8/site-packages/setuptools/_distutils/cmd.py", line 107, in __getattr__
          raise AttributeError(attr)
      AttributeError: cython_sources
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

[notice] A new release of pip is available: 23.0.1 -> 23.2
[notice] To update, run: python3 -m pip install --upgrade pip
The command '/bin/sh -c /app/venv/bin/pip --no-cache-dir install -r /app/requirements.txt' returned a non-zero code: 1
ERROR: Service 'mwdb' failed to build : Build failed
```

**Expected behaviour**

Sucessful build with mwdb running as a result.

**Reproduction Steps**

1. Download latest release version from [github](https://github.com/CERT-Polska/mwdb-core/releases/tag/v2.9.0)
2. Unzip
3. run `gen_vars.sh`
4. run `docker-compose up`
5. see error message above

Same error also reproducable with the 2.8.2 Version.


**Screenshots**

<!-- If applicable, add screenshots to help explain your problem. -->

**Additional context**

OS: Ubuntu 22.04.2 LTS
DockerVersion: 
```
Engine:
  Version:          20.10.21
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.1
  Git commit:       20.10.21-0ubuntu1~22.04.3
  Built:            Thu Apr 27 05:37:25 2023
  OS/Arch:          linux/amd64
  Experimental:     false
containerd:
  Version:          1.6.12-0ubuntu1~22.04.3
  GitCommit:        
runc:
  Version:          1.1.4-0ubuntu1~22.04.3
  GitCommit:        
docker-init:
  Version:          0.19.0
  GitCommit:        
```
Compose version: docker-compose version 1.29.2

Note2: 

Trying to bump the version up to `6.0.1` in the `requirements.txt` as recommended in the [issue](https://github.com/yaml/pyyaml/issues/724) in the pyYAML github  did solve the error for building the container, but upon startup another error appeared. see below for error message:

```sh
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: using the "epoll" event method
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: nginx/1.24.0
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: built by gcc 10.2.1 20210110 (Debian 10.2.1-6) 
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: OS: Linux 5.19.0-46-generic
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: getrlimit(RLIMIT_NOFILE): 1048576:1048576
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker processes
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 9
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 10
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 11
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 12
mwdb_1      | Waiting for DB to become operational
redis_1     | 1:C 20 Jul 2023 09:06:04.454 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
redis_1     | 1:C 20 Jul 2023 09:06:04.454 # Redis version=7.0.12, bits=64, commit=00000000, modified=0, pid=1, just started
redis_1     | 1:C 20 Jul 2023 09:06:04.454 # Warning: no config file specified, using the default config. In order to specify a config file use redis-server /path/to/redis.conf
postgres_1  | 
postgres_1  | PostgreSQL Database directory appears to contain a database; Skipping initialization
postgres_1  | 
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 13
postgres_1  | 2023-07-20 09:06:04.497 UTC [1] LOG:  starting PostgreSQL 15.3 (Debian 15.3-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
mwdb_1      | Configuring mwdb-core instance
postgres_1  | 2023-07-20 09:06:04.497 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1  | 2023-07-20 09:06:04.497 UTC [1] LOG:  listening on IPv6 address "::", port 5432
redis_1     | 1:M 20 Jul 2023 09:06:04.454 * monotonic clock: POSIX clock_gettime
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 14
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 15
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 16
postgres_1  | 2023-07-20 09:06:04.501 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
redis_1     | 1:M 20 Jul 2023 09:06:04.455 * Running mode=standalone, port=6379.
redis_1     | 1:M 20 Jul 2023 09:06:04.455 # Server initialized
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 17
postgres_1  | 2023-07-20 09:06:04.505 UTC [29] LOG:  database system was shut down at 2023-07-20 09:04:06 UTC
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 18
postgres_1  | 2023-07-20 09:06:04.510 UTC [1] LOG:  database system is ready to accept connections
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 19
redis_1     | 1:M 20 Jul 2023 09:06:04.455 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
mwdb-web_1  | 2023/07/20 09:06:05 [notice] 8#8: start worker process 20
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * Loading RDB produced by version 7.0.12
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * RDB age 118 seconds
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * RDB memory usage when created 0.82 Mb
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * Done loading RDB, keys loaded: 0, keys expired: 0.
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * DB loaded from disk: 0.000 seconds
redis_1     | 1:M 20 Jul 2023 09:06:04.456 * Ready to accept connections
mwdb_1      | INFO  [mwdb.configure] Configuration already initialized... skipping
mwdb_1      | INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
mwdb_1      | INFO  [alembic.runtime.migration] Will assume transactional DDL.
mwdb_1      | INFO  [mwdb.configure] Database already initialized... skipping
mwdb_1      | 
mwdb_1      | MWDB configured successfully!
mwdb_1      | 
mwdb_1      | Use 'mwdb-core run' to run the server.
mwdb_1      | 
mwdb_1      | [2023-07-20 09:06:06 +0000] [1] [INFO] Starting gunicorn 20.1.0
mwdb_1      | [2023-07-20 09:06:06 +0000] [1] [INFO] Listening at: http://0.0.0.0:8080 (1)
mwdb_1      | [2023-07-20 09:06:06 +0000] [1] [INFO] Using worker: sync
mwdb_1      | [2023-07-20 09:06:06 +0000] [36] [INFO] Booting worker with pid: 36
mwdb_1      | [2023-07-20 09:06:06 +0000] [37] [INFO] Booting worker with pid: 37
mwdb_1      | [2023-07-20 09:06:06 +0000] [38] [INFO] Booting worker with pid: 38
mwdb_1      | [2023-07-20 09:06:06 +0000] [36] [ERROR] Exception in worker process
mwdb_1      | Traceback (most recent call last):
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
mwdb_1      |     worker.init_process()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
mwdb_1      |     self.load_wsgi()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
mwdb_1      |     self.wsgi = self.app.wsgi()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
mwdb_1      |     self.callable = self.load()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
mwdb_1      |     return self.load_wsgiapp()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
mwdb_1      |     return util.import_app(self.app_uri)
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
mwdb_1      |     mod = importlib.import_module(module)
mwdb_1      |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
mwdb_1      |     return _bootstrap._gcd_import(name[level:], package, level)
mwdb_1      |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
mwdb_1      |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
mwdb_1      |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
mwdb_1      |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
mwdb_1      |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
mwdb_1      |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
mwdb_1      |   File "/app/mwdb/app.py", line 8, in <module>
mwdb_1      |     from mwdb.core.app import api, app
mwdb_1      | ModuleNotFoundError: No module named 'mwdb.core.app'
mwdb_1      | [2023-07-20 09:06:06 +0000] [36] [INFO] Worker exiting (pid: 36)
mwdb_1      | [2023-07-20 09:06:06 +0000] [39] [INFO] Booting worker with pid: 39
mwdb_1      | [2023-07-20 09:06:06 +0000] [37] [ERROR] Exception in worker process
mwdb_1      | Traceback (most recent call last):
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
mwdb_1      |     worker.init_process()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
mwdb_1      |     self.load_wsgi()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
mwdb_1      |     self.wsgi = self.app.wsgi()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
mwdb_1      |     self.callable = self.load()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
mwdb_1      |     return self.load_wsgiapp()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
mwdb_1      |     return util.import_app(self.app_uri)
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
mwdb_1      |     mod = importlib.import_module(module)
mwdb_1      |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
mwdb_1      |     return _bootstrap._gcd_import(name[level:], package, level)
mwdb_1      |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
mwdb_1      |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
mwdb_1      |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
mwdb_1      |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
mwdb_1      |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
mwdb_1      |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
mwdb_1      |   File "/app/mwdb/app.py", line 8, in <module>
mwdb_1      |     from mwdb.core.app import api, app
mwdb_1      | ModuleNotFoundError: No module named 'mwdb.core.app'
mwdb_1      | [2023-07-20 09:06:06 +0000] [37] [INFO] Worker exiting (pid: 37)
mwdb_1      | Traceback (most recent call last):
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 202, in run
mwdb_1      |     self.manage_workers()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 551, in manage_workers
mwdb_1      |     self.spawn_workers()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 623, in spawn_workers
mwdb_1      |     time.sleep(0.1 * random.random())
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
mwdb_1      |     self.reap_workers()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
mwdb_1      |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
mwdb_1      | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
mwdb_1      | 
mwdb_1      | During handling of the above exception, another exception occurred:
mwdb_1      | 
mwdb_1      | Traceback (most recent call last):
mwdb_1      |   File "/app/venv/bin/gunicorn", line 8, in <module>
mwdb_1      |     sys.exit(run())
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
mwdb_1      |     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 231, in run
mwdb_1      |     super().run()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 72, in run
mwdb_1      |     Arbiter(self).run()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 229, in run
mwdb_1      |     self.halt(reason=inst.reason, exit_status=inst.exit_status)
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 342, in halt
mwdb_1      |     self.stop()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 393, in stop
mwdb_1      |     time.sleep(0.1)
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
mwdb_1      |     self.reap_workers()
mwdb_1      |   File "/app/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
mwdb_1      |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
mwdb_1      | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
```

--- a/dev/bump_version.json
+++ b/dev/bump_version.json
@@ -4,7 +4,6 @@
         "mwdb/version.py": "app_version = \"$VERSION\"",
         "mwdb/web/package.json": "\"name\": \"mwdb-web\",\\s+\"version\": \"$VERSION\"",
         "mwdb/web/package-lock.json": "\"name\": \"mwdb-web\",\\s+\"version\": \"$VERSION\"",
-        "mwdb/web/src/commons/package.json": "\"version\": \"$VERSION\"",
         "docs/conf.py": "release = '$VERSION'",
         "docker-compose.yml": "image: certpl/mwdb(?:-web)?\\:v$VERSION"
     },

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -9,7 +9,7 @@ services:
     depends_on:
       - postgres
       - redis
-    image: certpl/mwdb:v2.9.0
+    image: certpl/mwdb:v2.9.1
     restart: on-failure
     env_file:
       # NOTE: use gen_vars.sh in order to generate this file
@@ -22,7 +22,7 @@ services:
     build:
       context: .
       dockerfile: deploy/docker/Dockerfile-web
-    image: certpl/mwdb-web:v2.9.0
+    image: certpl/mwdb-web:v2.9.1
     ports:
       - "80:80"
     restart: on-failure

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -21,7 +21,7 @@ copyright = '2021, CERT Polska'
 author = 'CERT Polska'
 
 # The full version, including alpha/beta/rc tags
-release = '2.9.0'
+release = '2.9.1'
 
 
 # -- General configuration ---------------------------------------------------

--- a/mwdb/version.py
+++ b/mwdb/version.py
@@ -4,5 +4,5 @@ try:
 except IOError:
     git_revision = ""
 
-app_version = "2.9.0"
+app_version = "2.9.1"
 app_build_version = f"{app_version}+{git_revision}" if git_revision else app_version

--- a/mwdb/web/package.json
+++ b/mwdb/web/package.json
@@ -1,6 +1,6 @@
 {
     "name": "mwdb-web",
-    "version": "2.9.0",
+    "version": "2.9.1",
     "private": true,
     "scripts": {
         "dev": "vite",

--- a/requirements.txt
+++ b/requirements.txt
@@ -17,7 +17,7 @@ luqum==0.7.4
 python-json-logger==2.0.2
 click==8.1.3
 click-default-group==1.2.2
-PyYAML==5.4
+PyYAML==6.0.1
 redis==4.5.4
 boto3==1.24.38
 typed-config==1.1.0

--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@ Under the hood of mwdb.cert.pl service hosted by CERT.pl.
 """
 
 setup(name="mwdb-core",
-      version="2.9.0",
+      version="2.9.1",
       description="MWDB Core malware database",
       long_description=LONG_DESCRIPTION,
       author="CERT Polska",

